package.json

```diff
@@ -44,7 +45,7 @@
     "compile": "tsc -p . && cp -r src/v2 build/src/ && cp -r proto* build/ && cp test/*.js build/test",
     "prepare": "npm run compile",
     "pretest": "npm run compile",
-    "proto": "mkdir -p proto && pbjs -t static-module -w commonjs -p protos google/bigtable/v2/bigtable.proto | pbts -o proto/bigtable.d.ts -"
+    "proto": "mkdir -p proto && pbjs -t static-module -w commonjs -p protos google/bigtable/v2/bigtable.proto | pbts -o proto/bigtable.d.ts - && pbjs -t static-module -w commonjs -p protos google/bigtable/admin/v2/bigtable_instance_admin.proto | pbts -o proto/bigtable_instance_admin.d.ts - && pbjs -t static-module -w commonjs -p protos google/bigtable/admin/v2/bigtable_table_admin.proto | pbts -o proto/bigtable_table_admin.d.ts -"
```

Comment:
This is a very big command! Could we either

1. Bundle the types together into a single file (this is untested but I think it works)
    ```sh
    $ mkdir -p proto && pbjs -t static-module -w commonjs -p protos google/bigtable/v2/bigtable.proto google/bigtable/admin/v2/bigtable_instance_admin.proto google/bigtable/admin/v2/bigtable_table_admin.proto | pbts -o proto/bigtable.d.ts -
    ```
1. Have separate commands per file

While we're at it lets install `mkdirp` as a dev dependency and use that instead of `mkdir -p`.

Reply:
I thought that might've been what you meant, but I guess I took the safe route - it works! Pushing now 👍